1_preprocessing_data/README.md

```diff
@@ -2,13 +2,37 @@
 
 In this module, we present our pipeline for preprocessing the NF1 Schwann Cell data.
 
-## Illumination Correction
+## Illumination Correction (IC)
 
 To correct for illumination issues within the NF1 data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
 We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
 
 Illumination correction is an important step in cell image analysis pipelines as it helps with downstream processes like segmentation (more accuracy in segmenting shape and identifying objects to segment) and feature extraction (accurate measurements in intensity, texture, etc.).
 
+Being able to visualize if images need IC can be very easy or very hard depending on the dataset. 
+In the NF1 dataset, the raw images are impossible to tell with the human eye that there are any illumination issues.
+
+To be able to see illumination issues within the images, we use Fiji to increase the maximum limits of the display range to brighten the image using the `B&C` function (Figure 1).
+
+![NF1_IC_Demonstration.png](example_figs/NF1_IC_Demonstration.png)
+> Figure 1. Use of Fiji to Determine Illumination Issues. This figure shows how increasing the maximum limit for the display range reveals illumination issues (e.g. vignetting like in the DAPI channel). The increasing of the upper limits for display starts to show where there is brighter areas in the image that was hidden when the upper limit was lower.
+
+The documentation regarding this function states:
```

Comment:
Maybe I missed it while reading it.

Would it be best to add the name of the function? Fiji is a well known program! Adding the name will make it easier for other readers/users to find which function you are talking about!